src/App.js

```diff
@@ -7,24 +7,31 @@ import AppNavBar from "./components/Navbar.js";
 import Loading from "./components/Loading.js";
 import AppTable from "./components/Table.js";
 
-const searchForArticles = async searchString => {
+import { useAnonymousLogin } from './auth'
+
+const searchForArticles = async (searchString, token) => {
   const host =
-    process.env.NODE_ENV === "production"
-      ? "https://us-central1-readme-arkiv.cloudfunctions.net/search"
-      : "http://localhost:5000/readme-arkiv/us-central1/search";
+  process.env.NODE_ENV === "production"
+  ? "https://us-central1-readme-arkiv.cloudfunctions.net/search"
+  : "http://localhost:5000/readme-arkiv/us-central1/search";
 
   try {
+    if (!token) {
+      throw new Error('No token, so why even bother?')
+    }
+
     const res = await fetch(
       `${host}?searchString=${encodeURIComponent(searchString)}`,
       {
         headers: {
+          Authorization: `Bearer ${token}`,
```

Comment:
Why do we add "Bearer" to the auth field if it's ignored when processed in the Firebase function?

Reply:
Nevermind, I didn't know about the [standard](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication)!